tests/test_wallet.py

```diff
@@ -0,0 +1,32 @@
+def test_total_balance(wallet, wallet_stub):
+    assert wallet.total_balance is not None
+
+
+def test_total_pending(wallet, wallet_stub):
+    assert wallet.total_pending is not None
+
+
+def test_accounts_when_has_accounts(wallet, account, wallet_stub):
+    accounts = wallet.accounts
+    assert len(wallet.accounts) == 1
```

Comment:
In wallet.accounts right now, it returns a list of 2 accounts by default. But since you're using the wallet_stub fixture, which uses the wallet_with_accounts.json file, which only has one account, this will pass, correct?

That said, wouldn't it be possible for a Wallet to have multiple accounts, and we wouldn't want to test that account length equals 1. Or are you keep it strict for the purpose of testing the wallet_stub fixture?

Reply:
wallet.accounts could return an empty list, or a list of N accounts, depending on how many are actually in your wallet.

The 2 accounts that are currently returned by wallet.accounts in `master` appear to be a bit of hard-coded dummy data, rather than what would actually be returned from the node.  I had to remove this line and make this property return the original `return self._get_accounts()` line so this test would pass.

Reply:
Sounds good! Yeah i figured it was dummy data, thought I would check that you'd taken that into account. Nice work on these tests!

Reply:
@jordancoil @crolfe Yes that was some leftover dummy data.